doc/rados/troubleshooting/log-and-debug.rst

```diff
@@ -200,59 +200,105 @@ to their default level or to a level suitable for normal operations.
 | ``objectcacher``   |     0     |      5       |
 +--------------------+-----------+--------------+
 | ``client``         |     0     |      5       |
 +--------------------+-----------+--------------+
-| ``osd``            |     0     |      5       |
+| ``osd``            |     1     |      5       |
 +--------------------+-----------+--------------+
 | ``optracker``      |     0     |      5       |
 +--------------------+-----------+--------------+
 | ``objclass``       |     0     |      5       |
 +--------------------+-----------+--------------+
-| ``filestore``      |     1     |      5       |
+| ``filestore``      |     1     |      3       |
 +--------------------+-----------+--------------+
-| ``journal``        |     1     |      5       |
+| ``journal``        |     1     |      3       |
 +--------------------+-----------+--------------+
 | ``ms``             |     0     |      5       |
 +--------------------+-----------+--------------+
 | ``mon``            |     1     |      5       |
 +--------------------+-----------+--------------+
-| ``monc``           |     0     |      5       |
+| ``monc``           |     0     |      10      |
 +--------------------+-----------+--------------+
-| ``paxos``          |     0     |      5       |
+| ``paxos``          |     1     |      5       |
 +--------------------+-----------+--------------+
 | ``tp``             |     0     |      5       |
 +--------------------+-----------+--------------+
 | ``auth``           |     1     |      5       |
 +--------------------+-----------+--------------+
-| ``finisher``       |     1     |      5       |
+| ``crypto``         |     1     |      5       |
++--------------------+-----------+--------------+
+| ``finisher``       |     1     |      1       |
++--------------------+-----------+--------------+
+| ``reserver``       |     1     |      1       |
 +--------------------+-----------+--------------+
 | ``heartbeatmap``   |     1     |      5       |
 +--------------------+-----------+--------------+
 | ``perfcounter``    |     1     |      5       |
 +--------------------+-----------+--------------+
 | ``rgw``            |     1     |      5       |
 +--------------------+-----------+--------------+
+| ``rgw sync``       |     1     |      5       |
++--------------------+-----------+--------------+
+| ``civetweb``       |     1     |      10      |
++--------------------+-----------+--------------+
 | ``javaclient``     |     1     |      5       |
 +--------------------+-----------+--------------+
 | ``asok``           |     1     |      5       |
 +--------------------+-----------+--------------+
-| ``throttle``       |     1     |      5       |
+| ``throttle``       |     1     |      1       |
++--------------------+-----------+--------------+
+| ``refs``           |     0     |      0       |
++--------------------+-----------+--------------+
+| ``xio``            |     1     |      5       |
```

Comment:
might want to remove the `kinetic` entry here. it's not built by default anymore.

Reply:
@tchaikov Ok, updated.